app/models/miq_server/role_management.rb

```diff
@@ -167,6 +167,15 @@ def assign_role(server_role, priority = nil)
     assigned_server_role
   end
 
+  LOGIN_ROLES = %w(user_interface web_services).freeze
```

Comment:
Note, after discussion with others, it makes sense to try to get the MiqUiWorker to require these roles web_services role since the UI needs to hit the rest api for authentication instead of having a hardcoded list of roles.  It would be nice if we enable only the user_interface role and the web_services roles was enabled as a dependency.  Will need to try that out and revise this.